src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -419,6 +570,45 @@ def builtin_cast(
     )
 
 
+def builtin_make_const_list(
+    transformer: "PythonTaskletCodegen", node: itir.Expr, node_args: list[itir.Expr]
+) -> list[ValueExpr]:
+    args = [transformer.visit(arg)[0] for arg in node_args]
+    assert all([isinstance(x, (SymbolExpr, ValueExpr)) for x in args])
+    args_dtype = set([x.dtype for x in args])
+    assert len(args_dtype) == 1
+    dtype = next(iter(args_dtype))
+
+    var_name = unique_var_name()
+    transformer.context.body.add_array(var_name, (len(args),), dtype, transient=True)
+    var_array = transformer.context.body.arrays[var_name]
+    var_node = transformer.context.state.add_access(var_name)
+
+    code = ""
+    tasklet_input_mapping = {}
+    for i, arg in enumerate(args):
+        rhs = "out" if len(args) == 1 else f"out[{i}]"
+        if isinstance(arg, SymbolExpr):
+            code += f"{rhs} = {arg.value}\n"
+        else:
+            connector = f"{arg.value.data}_v"
+            code += f"{rhs} = {connector}\n"
+            tasklet_input_mapping[connector] = arg.value
+
```

Comment:
If I understand that code correctly, you add multiple writes in a single Tasklet?
I am not saying that you should change it, but I do not like it.
Although it would blow up the SDFG, but you could generate individual Tasklets.
It would allow DaCe move them arround, because now it has to keep the lhs alive until all elements are computed.